src/modules/ui/nodes/UINodeCampaignSelector.cpp

```diff
@@ -42,16 +42,33 @@ void UINodeCampaignSelector::renderSelectorEntry (int index, const CampaignPtr&
 	if (!t || !t->isValid())
 		t = loadTexture("icon-campaign");
 
-	if (!data->getText().empty()) {
-		const BitmapFontPtr& font = getFont(HUGE_FONT);
-		const int textHeight = font->getTextHeight(data->getText());
-		const int fontX = std::max(x, x + colWidth / 2 - font->getTextWidth(data->getText()) / 2);
-		const int fontY = y + rowHeight - textHeight - 1;
-		renderFilledRect(x, fontY - 1, colWidth - _padding, textHeight + 2, colorBlack);
-		renderImage(t, x, y, colWidth, rowHeight - textHeight, alpha);
-		font->printMax(data->getText(), colorWhite, fontX, fontY, colWidth);
-	} else {
-		renderImage(t, x, y, colWidth, rowHeight, alpha);
+	if (isSmallScreen()) {
+		if (!data->getText().empty()) {
+			const BitmapFontPtr& font = getFont(HUGE_FONT);
+			const int textHeight = font->getTextHeight(data->getText());
+			const int fontX = std::max(x, x + colWidth / 2 - font->getTextWidth(data->getText()) / 2);
+			const int fontY = y + rowHeight - textHeight - 1;
+			renderFilledRect(x, fontY - 1, colWidth - _padding, textHeight + 2, colorBlack);
+			renderImage(t, x, y, colWidth, rowHeight - textHeight, alpha);
+			font->printMax(data->getText(), colorWhite, fontX, fontY, colWidth);
+		} else {
+			renderImage(t, x, y, colWidth, rowHeight, alpha);
+		}
+	}else{
+		const int marginX = 35, padding = 20, marginXtotal = 2 * marginX + padding;
```

Comment:
please don't use absolute numbers here - this means it works on your current resolution - but not on others